ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -140,36 +137,31 @@ QtObject {
     }
 
     function createCommunityCategory(communityId, categoryName, channels) {
-        // Not Refactored Yet
-//        chatsModelInst.communities.createCommunityCategory(communityId, categoryName, channels);
+        communitiesModuleInst.createCommunityCategory(communityId, categoryName, channels);
     }
 
     function editCommunityCategory(communityId, categoryId, categoryName, channels) {
-        // Not Refactored Yet
-//        chatsModelInst.communities.editCommunityCategory(communityId, categoryId, categoryName, channels);
+        communitiesModuleInst.editCommunityCategory(communityId, categoryId, categoryName, channels);
     }
 
-    function deleteCommunityCategory(categoryId) {
-        // Not Refactored Yet
-//        chatsModelInst.communities.deleteCommunityCategory(chatsModelInst.communities.activeCommunity.id, categoryId);
+    function deleteCommunityCategory(communityId, categoryId) {
+        communitiesModuleInst.deleteCommunityCategory(communityId, categoryId);
```

Comment:
@osmaczko `deleteCommunityCategory ` should not be part of `communitiesModuleInst`.